CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing
```

Comment:
Well organization's name would be best to sort the table because:-
***
_If the table is sorted according to org's name._

Organization | Project | Author |
:------------ | :------------- | :-------------
Helm | Random Project | Raghav
Kubernetes | Project One | Nikhita
Kubernetes | Question Two | Random
Microsoft | Visual Studio Code | Nikhita
Teammates | Coverage Hero | Amar
Teammates | Student Data | Vamsi

All the project of the same organization will be in sequence, easy for a user to find org or project. :wink: 
***
_If the table is sorted according to project's name._

Organization | Project | Author |
:------------ | :------------- | :-------------
Kubernetes | Project One | Nikhita
Helm | Random Project | Raghav
Teammates | Student Data | Vamsi
Microsoft | Visual Studio Code | Nikhita
Teammates | Coverage Hero | Amar
Kubernetes | Question Two | Random

But in this case, nothing looks sorted :stuck_out_tongue_closed_eyes: 
***
Your thoughts?